docs/source/extensions/logging.rst

```diff
@@ -14,56 +14,85 @@
 Logging
 #######
 
-Lightning supports the most popular logging frameworks (TensorBoard, Comet, etc...).
```

Comment:
I liked the statement as it emphasis on our good support for loggers

Reply:
we have them right below: https://140783-178626720-gh.circle-artifacts.com/0/html/extensions/logging.html